### PR TITLE
For #3865: Use separate app name and icon for Nightly build

### PR DESCRIPTION
**Pull request by colintheshots**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: app/src/nightly/res/values/static_strings.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Name of the application -->
+    <string name="app_name" translatable="false">Firefox Preview Nightly</string>
```
[the remark this answers is not in the document]
Comment (colintheshots) [reply app/src/nightly/res/values/static_strings.xml]: Do you have a suggestion? The issue in question was about having a different app name and icon for Nightly. Calling it just "Preview Nightly" would make it hard to find alphabetically in the app drawer. "Firefox Nightly" would be too similar to Fennec.

[the remark this answers is not in the document]
Comment (colintheshots) [reply app/src/nightly/res/values/static_strings.xml]: Would "Fenix Nightly" be better? We can call it an easier to find name in the Play Store like "Firefox Preview Nightly", but users may be unable to find it in the app drawer.

[the remark this answers is not in the document]
Comment (colintheshots) [reply app/src/nightly/res/values/static_strings.xml]: @jhugman ?

[the remark this answers is not in the document]
Comment (jhugman) [reply app/src/nightly/res/values/static_strings.xml]: If it were up to me, `Firefox Nightly` or `Fenix Nightly` would be my preference.

As you say, app drawer findability is important, so perhaps `Firefox Nightly` (it has a different icon to Fennec Nightly) might be the better choice.

I appreciate the thought on a11y. Is it a reasonable assumption that if you're on a nightly, you know what Fenix is? i.e. use `Fenix Nightly`.

Pick one of these, and see who shouts.

**Pull request merged**